docs/async.md

```diff
@@ -10,6 +10,24 @@ long-lived network connections such as WebSockets.
 If you're working with an async web framework then you'll also want to use an
 async client for sending outgoing HTTP requests.
 
+## Enabling Async support
+
+To enable async support you'll need to install some additional dependencies:
+
+If you're using Python's [standard `asyncio` support](https://docs.python.org/3/library/asyncio.html) then:
+
+```shell
+$ pip install httpx['asyncio']
```

Comment:
Should we quote this so zsh users do not encounter an error? e.g. 

```
❯ pip install httpx['asyncio']
zsh: no matches found: httpx[asyncio]
```

Reply:
I don't know, maybe? Could you show me the `±suggestion`?
We're probably using this style in a few places, right?

Reply:
```suggestion
$ pip install "httpx[asyncio]"
```

Reply:
We haven't released this.

Reply:
`pip` doesn't fail on unknown extras.

It fails with a parse error

```
    raise InvalidRequirement(
pip._vendor.packaging.requirements.InvalidRequirement: Parse error at ""['asynci"": Expected string_end
```